search_roadmap_export/search_roadmap_export.py

```diff
@@ -0,0 +1,122 @@
+"""
+This script pulls selected tasks from the CC Search Roadmap Asana project, then
+fills that information into a lektor databag, and pushes it to a file in
+creativecommons/creativecommons.github.io-source.
+"""
+
+# Standard Library
+import os
+import json
+
+# Third-party
+import asana
+from github import Github
+
+ASANA_CLIENT = asana.Client.access_token(os.environ["ADMIN_ASANA_TOKEN"])
+ASANA_GIDS = { # GIDs are unique IDs for Asana resources, each task, tag, project, section, etc. has one.
+    'ROADMAP_SECTIONS': {
```

Comment:
do we have to hardcode these? we'll need to update them every time a new quarter is added. can't we just get sections matching the pattern?

Reply:
Nope, did some more digging and with some regex magic we now only need one GID (for the roadmap project itself). No need for GID maintenance.

When I say that we "need" the roadmap project GID, we don't technically. We would go through a semi-lengthy process every run but just hard coding the GID (which as far as I know, never changes) is far faster and simpler.